photon_installer/installer.py

```diff
@@ -88,12 +89,12 @@ class Installer(object):
     linux_dependencies = ["devel", "drivers", "docs", "oprofile", "dtb"]
 
     def __init__(self, working_directory="/mnt/photon-root",
-                 rpm_path=os.path.dirname(__file__)+"/../stage/RPMS", log_path=os.path.dirname(__file__)+"/../stage/LOGS",
```

Comment:
I think this will break imagebuilder as we pass "rpm_path" as argument from imagebuilder